### ISSUE TITLE
The system cannot find the path specified.

### ISSUE THREAD

**Issue by rikbrowning** [cyco130/vavite#13]
Node: v16.17.0
vite: v3.0.9
vavite: v1.4.1
Windows 11

Had a brief look at the generated powershell files for vavite and it seems there is a difference between vavite and my other installed commands.
vavite:
```ps1
#!/usr/bin/env pwsh
$basedir=Split-Path $MyInvocation.MyCommand.Definition -Parent

$exe=""
if ($PSVersionTable.PSVersion -lt "6.0" -or $IsWindows) {
  # Fix case when both the Windows and Linux builds of Node
  # are installed in the same directory
  $exe=".exe"
}
$ret=0
if (Test-Path "$basedir//bin/env$exe") {
  # Support pipeline input
  if ($MyInvocation.ExpectingInput) {
    $input | & "$basedir//bin/env$exe" node "$basedir/../@vavite/multibuild-cli/cli.js" $args
  } else {
    & "$basedir//bin/env$exe" node "$basedir/../@vavite/multibuild-cli/cli.js" $args
  }
  $ret=$LASTEXITCODE
} else {
  # Support pipeline input
  if ($MyInvocation.ExpectingInput) {
    $input | & "/bin/env$exe" node "$basedir/../@vavite/multibuild-cli/cli.js" $args
  } else {
    & "/bin/env$exe" node "$basedir/../@vavite/multibuild-cli/cli.js" $args
  }
  $ret=$LASTEXITCODE
}
exit $ret
```

vite
```ps1
#!/usr/bin/env pwsh
$basedir=Split-Path $MyInvocation.MyCommand.Definition -Parent

$exe=""
if ($PSVersionTable.PSVersion -lt "6.0" -or $IsWindows) {
  # Fix case when both the Windows and Linux builds of Node
  # are installed in the same directory
  $exe=".exe"
}
$ret=0
if (Test-Path "$basedir/node$exe") {
  # Support pipeline input
  if ($MyInvocation.ExpectingInput) {
    $input | & "$basedir/node$exe"  "$basedir/../vite/bin/vite.js" $args
  } else {
    & "$basedir/node$exe"  "$basedir/../vite/bin/vite.js" $args
  }
  $ret=$LASTEXITCODE
} else {
  # Support pipeline input
  if ($MyInvocation.ExpectingInput) {
    $input | & "node$exe"  "$basedir/../vite/bin/vite.js" $args
  } else {
    & "node$exe"  "$basedir/../vite/bin/vite.js" $args
  }
  $ret=$LASTEXITCODE
}
exit $ret
```

Seems that on windows it is trying to use the bin/env for vavite yet for the vite command it is correctly using node.

**Comment by cyco130** [comment]
Thanks for reporting. I don't know how many times I had issues with Node binaries on Windows :) I'll have a look on the weekend.

**Comment by rikbrowning** [comment]
No worries. Honestly I've no idea why only this package is doing it. Looks to me like everything is set up correct so why npm generates a different script is very strange.

**Comment by cyco130** [comment]
Fixed in 1.4.2 (just released) 🥳
